SPECS/glib/glib.spec

```diff
@@ -69,18 +71,19 @@ The glib2-doc package includes documentation for the GLib library.
 
 %prep
 %setup -q
-%patch0 -p1
-%patch1 -p1
 
 %build
-./autogen.sh
-%configure --with-pcre=system \
-            --enable-static \
-            --enable-gtk-doc
-make %{?_smp_mflags}
+%meson \
```

Comment:
Double-checking: are we still keeping the equivalent of the `--with-pcre` switch from the earlier version?

Reply:
Yes, the new version of glib is using the system version of pcre by default